README.dev.md

```diff
@@ -15,8 +15,25 @@
       - Make sure to include the sha256 digest for each image, which [ensures that each image pull is immutable](https://renovatebot.com/docs/docker/#digest-pinning). Use `docker inspect --format='{{index .RepoDigests 0}}' $IMAGE` to get the digest.
 
 - Open a PR and wait for buildkite to pass and for your changes to be approved, then merge and check out master.
-- Test what is currently checked in to master by [installing](docs/install.md) Sourcegraph on fresh cluster.
-- Create a git tag and push it to the repository:
+- Test what is currently checked in to master by [installing](docs/install.md) Sourcegraph on a fresh cluster:
+  - Provision a new cluster:
+    1.  Create a cluster unique name that is identifiable to you (e.g `ggilmore-test`) in the [Sourcegraph Auxiliary GCP Project](https://console.cloud.google.com/kubernetes/list?project=sourcegraph-server&organizationId=1006954638239). You can create pool that has `3` nodes, each with `8` vCPUs and `30` GB memory (for a total of `24` vCPUs and `90` GB memory).
+        - See this screenshot, but note that the UI could have changed: ![](https://imgur.com/RuCyGX2.png)
+    1.  It’ll take a few minutes for it to be provisioned, you’ll see a green checkmark when it is done.
+    1.  Click on the `connect` button next to your cluster, it’ll give you a command to copy+paste in your terminal.
+    1.  Run the command in your terminal. Once it finishes, run `kubectl config current-context`. It should tell you that it’s set up to talk to the cluster that you just created.
+    1.  In order to give yourself permissions to create roles on the cluster, run: `kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user $YOUR_NAME@sourcegraph.com`
```

Comment:
Should there be a step here for port forwarding?:
```suggestion
    1.  In order to give yourself permissions to create roles on the cluster, run: `kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user $YOUR_NAME@sourcegraph.com`
    1. Make the frontend port accessible at localhost:30080 by running `kubectl port-forward $(kubectl get pod -l app=sourcegraph-frontend -o template --template="{{(index .items 0).metadata.name}}") 30080:3080`
```

Reply:
I decided to just link to install.md which has these quickstart instructions